clap_builder/src/builder/arg.rs

```diff
@@ -1739,8 +1739,8 @@ impl Arg {
 
     /// Value for the argument when the flag is present but no value is specified.
     ///
-    /// This configuration option is often used to give the user a shortcut and allow them to
-    /// efficiently specify an option argument without requiring an explicitly value. The `--color`
+    /// Configure a command-line argument that gives the user a shortcut and allows them to
```

Comment:
> command-line argument

"command-line" is implied from context of this library and makes this overly verbose.

"option" is more specific than "argument" as this isn't as useful for positional arguments.